server/i18n/en.json

```diff
@@ -2630,15 +2630,15 @@
   },
   {
     "id": "api.post.disabled_all",
-    "translation": "@all has been disabled because the channel has more than {{.Users}} users."
+    "translation": "@all notifications are disabled in channels with more than {{.Users}} users to maintain server performance. Contact your system administrator for more information."
   },
   {
     "id": "api.post.disabled_channel",
-    "translation": "@channel has been disabled because the channel has more than {{.Users}} users."
+    "translation": "@channel notifications are disabled in channels with more than {{.Users}} users to maintain server performance. Contact your system administrator for more information."
```

Comment:
this doesn't seem to be getting applied. see screenshot
<img width="597" alt="image" src="https://github.com/user-attachments/assets/0d090dfd-2565-420a-8377-d76dafa222ec" />